### ISSUE TITLE
How to use mapbox styles？

### ISSUE THREAD

**Issue by tangzijun** [protomaps/basemaps#84]
In addition to using several themes in basempa, how can I customize my own theme? For example, when I use existing mapbox styles or openmaptiles styles, the display of pmtiles is abnormal. How can I modify the style of mapbox、openmaptiles, or is there a conversion tool? 

<img width="633" alt="image" src="https://github.com/protomaps/basemaps/assets/11813048/21a0e250-9e34-4e4b-9816-43d091246a4e">

**Comment by tordans** [comment]
This will not work out of the box and even if it does, you would have to **check licenses** if you are allowed to use the styles in this way. 

The way the vector tiles data and vector style interact is: 
- The vector tiles data have to include the same objects in the same "form" (simplification per zoom level).
- The objects need to have the same attributes
Then, the style can use those attributes to style the object.
This is true regardless of how you store the vector tiles data, so it is not PMTile specific.

TL/DR: You cannot just use other styles.

**Comment by bdon** [comment]
@tordans is correct - the software in this repository is designed to only produce and consume the protomaps tile definitions.  These are under development but generally documented here:

https://protomaps.com/docs/frontends/basemap-layers
https://tilezen.readthedocs.io/en/latest/layers/

If you are using different tileset data tools, you can still use PMTiles, use the go-pmtiles tool to convert.
